tests/test_OBD.py

```diff
@@ -122,30 +122,30 @@ def test_port_name():
     """
     o = obd.OBD("/dev/null")
     o.interface = FakeELM("/dev/null")
-    assert o.port_name() == o.interface._portname
+    assert o.get_port_name() == o.interface.get_port_name()
 
     o.interface = FakeELM("A different port name")
-    assert o.port_name() == o.interface._portname
+    assert o.get_port_name() == o.interface.get_port_name()
```

Comment:
I suppose it just comes down to preference; I'd prefer to leave these the way they were. I'd argue short is usually better. Also, if we decide to change them, lets do that in a separate PR.

Reply:
Shorter is never better unless there is a concern for memory, which there is not. More descriptive is better IMO.